library/kotlin/src/io/envoyproxy/envoymobile/Engine.kt

```diff
@@ -0,0 +1,17 @@
+package io.envoyproxy.envoymobile
+
+/**
+ * The engine for downstream clients to integrate Envoy Mobile
+ */
+interface Engine {
+
+  /**
+   *  @return a {@link StreamClient} instance
+   */
+  fun getStreamClient(): StreamClient
+
+  /**
+   *  @return a {@link StatsClient} instance
+   */
+  fun getStatsClient(): StatsClient
```

Comment:
would it make sense to simply name these `streamClient()` and `statsClient()`?

Reply:
@goaway ^

Reply:
I'm fine with that.